sycl/plugins/unified_runtime/ur/adapters/hip/usm_p2p.cpp

```diff
@@ -0,0 +1,31 @@
+//===--------- usm_p2p.cpp - HIP Adapter---------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===---------------------------------------------------------------===//
+
+#include "common.hpp"
+
+UR_APIEXPORT ur_result_t UR_APICALL
+urUsmP2PEnablePeerAccessExp(ur_device_handle_t, ur_device_handle_t) {
+  detail::ur::die(
+      "urUsmP2PEnablePeerAccessExp is not implemented for HIP adapter.");
+  return UR_RESULT_ERROR_UNSUPPORTED_FEATURE;
+}
+
+UR_APIEXPORT ur_result_t UR_APICALL
+urUsmP2PDisablePeerAccessExp(ur_device_handle_t, ur_device_handle_t) {
+  detail::ur::die(
+      "urUsmP2PDisablePeerAccessExp is not implemented for HIP adapter.");
+  return UR_RESULT_ERROR_UNSUPPORTED_FEATURE;
+}
+
+UR_APIEXPORT ur_result_t UR_APICALL
+urUsmP2PPeerAccessGetInfoExp(ur_device_handle_t, ur_device_handle_t,
+                             ur_exp_peer_info_t, size_t, void *, size_t *) {
+  detail::ur::die(
```

Comment:
This isn't the biggest deal, but this is changing the current behaviour which was to return false for all queries. You could make it do the same thing as it does in the level_zero adapter for instance: https://github.com/intel/llvm/blob/9e6f7962876c027e801608d43fd25f88ac3f81d6/sycl/plugins/unified_runtime/ur/adapters/level_zero/usm_p2p.cpp#L42

Reply:
Changed that.